src/main/java/org/isf/utils/db/DbSingleConn.java

```diff
@@ -33,10 +33,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.mysql.jdbc.exceptions.jdbc4.CommunicationsException;
+import com.mysql.cj.jdbc.exceptions.CommunicationsException;
```

Comment:
Here the hotfix was:
```
-import com.mysql.cj.jdbc.exceptions.CommunicationsException;
+import com.mysql.jdbc.exceptions.jdbc4.CommunicationsException;
```

but it should temporary work in both ways I think...

Reply:
I'm going to leave it like it is; when we worry about JDBC jar versions we can visit if needed.

Reply:
ok